packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -36,6 +36,81 @@ import 'viewport.dart';
 /// See also:
 ///
 ///  * [PageView], which is the widget this object controls.
+///
+/// {@tool sample}
+///
+/// This widget introduces a [PageView] with two pages using the default constructor. Both pages contain a [RaisedButton] allowing you
+/// to animate the [PageView] using a [PageController].
+///
+/// ```dart
+/// class MyPageView extends StatefulWidget {
+///   MyPageView({Key key}) : super(key: key);
+///
+///   _MyPageViewState createState() => _MyPageViewState();
+/// }
+///
+/// class _MyPageViewState extends State<MyPageView> {
+///   PageController _pageController;
+///
+///   @override
+///   void initState() {
+///     super.initState();
+///     _pageController = PageController();
+///   }
+///
+///   @override
+///   void dispose() {
+///     _pageController.dispose();
+///     super.dispose();
+///   }
+///
+///   @override
+///   Widget build(BuildContext context) {
+///     return PageView(
+///       controller: _pageController,
+///       children: [
+///         Container(
+///           color: Colors.red,
+///           child: Center(
+///             child: RaisedButton(
+///               color: Colors.white,
+///               onPressed: () {
+///                 if (_pageController.hasClients) {
+///                   _pageController.animateToPage(
+///                     1,
+///                     duration: const Duration(milliseconds: 400),
+///                     curve: Curves.easeInOut,
+///                   );
+///                 }
+///               },
+///               child: Text('Next'),
+///             ),
+///           ),
+///         ),
+///         Container(
+///           color: Colors.blue,
+///           child: Center(
+///             child: RaisedButton(
+///               color: Colors.white,
+///               onPressed: () {
+///                 if (_pageController.hasClients) {
+///                   _pageController.animateToPage(
+///                     0,
+///                     duration: const Duration(milliseconds: 400),
+///                     curve: Curves.easeInOut,
+///                   );
+///                 }
+///               },
+///               child: Text('Previous'),
+///             ),
+///           ),
+///         ),
+///       ],
+///     );
```

Comment:
```suggestion
///      ),
///     );
```
Additional closing parentheses for the addition of the `Directionality` widget.